Mobile/starter_project_dartsmiths/lib/onboarding/presentation/screen/onboarding.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+
+import '../../../core/utils/colors.dart';
+import '../../../core/utils/images.dart';
+import '../widgets/onboarding_box.dart';
+import '../widgets/onboarding_button.dart';
+import '../widgets/slide_dots.dart';
+
+class Onboarding extends StatefulWidget {
+  const Onboarding({super.key});
+
+  @override
+  State<Onboarding> createState() => _OnboardingState();
+}
+
+class _OnboardingState extends State<Onboarding> {
+  int index = 0;
+  static const pageTitle = "Read the article you want instantly";
+  static const pageDescription =
+      "You can read thousands of articles on Blog Club, save them in the application and share them with your loved ones.";
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        backgroundColor: scaffoldBackground,
+        body: Column(
+          children: [
+            SizedBox(
+              height: MediaQuery.of(context).size.height * 0.05,
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.all(6.0),
+                  child: Column(
+                    children: [
+                      Row(
+                        children: [
+                          OnboardingBox(
+                            imageUrl: onBoardingPhoto1,
+                            isLarge: false,
+                          ),
+                          OnboardingBox(
```

Comment:
it is more like a description box right? rename to OnboardingDescriptionBox or OnboardingStepperBox